### ISSUE TITLE
Centreon 2.8.18 Blank page after install

### ISSUE THREAD

**Issue by LastTrainHome** [centreon/centreon#6092]
Hello everyone,

I'm a newbie so I'm pretty lost
I am actually having an issue with Centreon after the install. The webpage **http://127.0.0.1/centreon/install/setup.php** return a "blank" page. [https://i.imgur.com/zLGhZfC.png](url)

I'm actually using Ubuntu 16.04.

The error.log of Apache2 return 

```
[Tue Feb 27 14:18:45.921815 2018] [:error] [pid 2839] [client 127.0.0.1:40728] PHP Notice:  Undefined index: centreon_engine_lib in /usr/local/centreon/www/install/steps/functions.php on line 243, referer: http://127.0.0.1/centreon/install/setup.php

[Tue Feb 27 14:18:45.921838 2018] [:error] [pid 2839] [client 127.0.0.1:40728] PHP Notice:  Undefined index: centreonbroker_cbmod in /usr/local/centreon/www/install/steps/functions.php on line 244, referer: http://127.0.0.1/centreon/install/setup.php

[Tue Feb 27 14:18:45.925757 2018] [:error] [pid 2839] [client 127.0.0.1:40728] PHP Warning:  preg_replace(): The /e modifier is no longer supported, use preg_replace_callback instead in /usr/local/centreon/GPL_LIB/Smarty/libs/Smarty_Compiler.class.php on line 270, referer: http://127.0.0.1/centreon/install/setup.php
```

Thanks for your help and ask me for details if you need some.

**Comment by lpinsivy** [comment]
Hi,

How did you install Centreon?

What is your version of PHP?

Regards,

**Comment by stemid** [comment]
I'm having the same issue.

Installed as root with install.sh script.

I set all my paths to /usr/local/centreon2 because I had an old centreon version in /usr/local/centreon. So every path suggested I added the number 2.

I did run php7 but downgrading to 5.6 fixed the problem. The requirements clearly state that only 5.x is supported. This is of course an issue on Debian Stretch but that topic has already been brought up elsewhere.
